### DISCUSSION TITLE
Enhance update command to only install patch level updates

### DISCUSSION THREAD

**Discussion by MajPay** [composer/composer#11441] [General]
Hi!

I was wondering if there are any plans to enhance the update command with a --patch-level-only (example name) flag.

Why would i want something like that?

I know i could use "~1.0.0" to make sure only the patch-level will be updated but i don't want to exclude minor updates from the regular update command. I would need the --patch-level-only flag for automation (CI pipelines). What i am looking for is a way to automatically install updates on a daily basis without risking the app to fail.

minor and major updates often introduce breaking changes that require migrations. Minor updates should not do that but sometimes they do, while patch-level updates won't change any API in 99.9% of the time.

**Comment by Seldaek** [comment]
Moved to https://github.com/composer/composer/issues/11446 - sounds doable for sure but not sure how easy exactly. But no we do not have plans for this currently, so PR welcome :D
